src/transformStateWithClones.js

```diff
@@ -8,6 +8,35 @@
  */
 function transformStateWithClones(state, actions) {
   // write code here
+  const states = [];
+  const newState = Object.assign({}, state);
+
+  for (const action of actions) {
+    switch (action.type) {
+      case 'addProperties' :
```

Comment:
Add cases in const.
For example const ACTION_TYPES = {
add: 'addProperties',
}